plugins/vlc-video/vlc-video-source.c

```diff
@@ -399,6 +399,8 @@ static void add_file(struct vlc_source *c, struct darray *array,
 		new_media = create_media_from_file(path);
 
 	if (new_media) {
+		libvlc_media_add_option_(new_media, ":network-caching=100");
```

Comment:
This should really be an option. 100ms is to reduce buffering on a local network device.